docs/user/openstack/README.md

```diff
@@ -216,16 +162,25 @@ Create and associate a floating IP to the ingress port:
 openstack floating ip create --port <ingress port id> <external network>
 ```
 
-### Without Floating IPs
```

Comment:
This was specifically targed towards people who were running providern-network clusters

Reply:
how common do you think it will be for real-life deployments? Perhaps we could mention it in the developers documentation instead -- I feel it's out of place in the user doc.

Reply:
I dont know to be honest, Tomas would know more than I do. Maybe we can add a thing for it in the dev docs later.